assets/js/base/components/notice-banner/style.scss

```diff
@@ -0,0 +1,116 @@
+.wc-block-components-notice-banner {
+	display: flex;
+	align-items: stretch;
+	align-content: flex-start;
+	color: $gray-800;
+	padding: $gap !important;
+	gap: $gap-small;
+	margin: $gap 0;
+	border-radius: 4px;
+	border-color: $gray-800;
+	font-weight: 400;
+	line-height: 1.5;
+	border: 1px solid;
+	@include font-size(small);
+	background-color: #fff;
+	box-sizing: border-box;
+
+	&:last-child {
+		margin-bottom: 0;
+	}
+
+	> .wc-block-components-notice-banner__content {
+		padding-right: $gap;
+		align-self: center;
+		white-space: normal;
+
+		> div *:first-child {
+			margin-top: 0;
+		}
+
+		> div *:last-child {
+			margin-bottom: 0;
+		}
+
+		&:last-child {
+			padding-right: 0;
+		}
+
+		.wc-block-components-notice-banner__summary {
+			margin: 0 0 $gap-smaller;
+			font-weight: 600;
+		}
+
+		ul,
+		ol {
+			margin: 0 0 0 $gap-large;
+			padding: 0;
+		}
+	}
+
+	> svg {
+		fill: #fff;
+		border-radius: 50%;
+		padding: 2px;
+		background-color: $gray-800;
+		flex-shrink: 0;
+	}
+
+	> .wc-block-components-button {
```

Comment:
Are there more buttons besides `.wc-block-components-notice-banner__dismiss`? If not, could we use that class name instead?

Also, I noticed that in TT3, with _Whisper_ styling, the button gets a weird border. Wondering if we should add some CSS resets to prevent themes messing with the style:

![imatge](https://user-images.githubusercontent.com/3616980/227496517-07aa8771-12f7-45eb-9b62-1a658480fb36.png)

Reply:
Some core notices do ("view more" links). I will add the reset though!